definitions/ext-gatsby_build/definition.yml

```diff
@@ -0,0 +1,20 @@
+domain: EXT
+# Derived from the Fastly log ingestion, a datacenter is a Point of Presence
```

Comment:
The comment is not relevant for this type, right?

Reply:
Oh yes good call, I had copied this from a template :D